SU-SWS/ace-sdssgryphon
# SDSS-276: footer headings

# READY FOR REVIEW

# Summary
- Setting the headings to white in the footer.
- Setting the colors of linked headers

# Review By (Date)
- 4/10/2023

# Urgency
- How critical is this PR?
- Normal

# Steps to Test

1. checkout this branch.
2. Clear caches.
3. Add `h1-h5` in the first footer content block. Add a link to one of them and add an external link on one. 
4. Verify the colors are correct. 

# Affected Projects or Products
- Does this PR impact any particular projects, products, or modules?

# Associated Issues and/or People
[- SDSS-276](https://stanfordits.atlassian.net/browse/SDSS-276)

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
